src/containers/audio-rewards-page/components/SendInputBody.tsx

```diff
@@ -164,25 +215,51 @@ const SendInputBody = ({
     [addressError, setAddressError, setDestinationAddress]
   )
 
+  const useSolSPLAudio = getRemoteVar(BooleanKeys.USE_SPL_AUDIO) as boolean
```

Comment:
note to self, bummer we have to do these casts - is this to avoid the null case? should fix this at some later time